app/process/handler/consensus-message/consensus-message.go

```diff
@@ -85,25 +85,25 @@ func (cmh ConsensusMessageHandler) handlePayload(payload []byte) error {
 		return errors.New(fmt.Sprintf("[%s] - Address is not valid - [%s]", m.TransactionId, address.String()))
 	}
 
-	messages, err := cmh.repository.GetTransaction(m.TransactionId, m.Signature)
+	messages, err := cmh.repository.GetTransaction(m.TransactionId, hex.EncodeToString(ethSig))
```

Comment:
maybe we can extract a variable `hexEncodedEthSig` from `hex.EncodeToString(ethSig)`. We are using it in 4-5 different places